src/main/java/com/minecolonies/coremod/network/messages/PermissionsMessage.java

```diff
@@ -95,10 +98,10 @@ public void toBytes(@NotNull final ByteBuf buf)
      */
     public static class Permission extends AbstractMessage<Permission, IMessage>
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Reduce this class from 63 to the maximum allowed 30 or externalize it in a public class. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS2972)